grasp_generation/scripts/eval_grasp_config_dict.py

```diff
@@ -62,15 +62,17 @@ class EvalGraspConfigDictArgumentParser(Tap):
 
 def compute_joint_angle_targets(
     args: EvalGraspConfigDictArgumentParser,
-    hand_pose_array: List[torch.Tensor],
-    grasp_orientations_array: List[torch.Tensor],
+    hand_pose: torch.Tensor,
+    grasp_orientations: torch.Tensor,
 ) -> np.ndarray:
     device = torch.device(f"cuda:{args.gpu}" if torch.cuda.is_available() else "cpu")
```

Comment:
Nit; Let's just use the device of hand_pose and instead of passing in args, just pass in hand_model_type